src/components/molecules/SettingTools.tsx

```
@@ -0,0 +1,85 @@
+                                        return serverName
+                                    }
+                                })
+                            }}>
+                            <div>{serverName}</div>
+                            {serverName !== openDescription ? <ChevronRight /> : <ChevronDown />}
+                        </div>
+                    </div>
+                    {serverName === openDescription && <div className="mt-2 ml-8">
+                        {toolDescriptions.map(t => (
+                            <div className="cursor-default py-3 px-6 mb-2 border border-neutral-300 rounded-md">
+                                <div className="mb-2">{t.name}</div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)